db/openshift/postgresql-deploy.dev.param

```diff
@@ -0,0 +1,21 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: ss-db
+# Template File: templates/postgresql-deploy.yaml
+#=========================================================
+# NAME=postgresql
+# APP_NAME=sheriff-scheduling
+# ROLE=db
+# IMAGE_NAMESPACE=tamjni-tools
```

Comment:
tamjni? I think our namespace is jag-shuber. Or is this openshift config just from the previous project?

Reply:
It's from FPO I think.

Reply:
`tamjni` is Traffic.  `jag-shuber` for Sheriff.

Reply:
@WadeBarnes high level, what are you hoping for with the openshift config? Start from the existing jag-shuber scripts, or start from another project like traffic?

Reply:
Good reference (structure wise) here; https://github.com/bcgov/orgbook-configurations

Reply:
Traffic has the Vue / .NetCore docker and OCP bits all worked out to support KC with the jag proxies; https://github.com/bcgov/violation-dispute-form

Reply:
Supreme court viewer uses the same Vue / .netCore tech stack we'll be using; https://github.com/bcgov/supreme-court-viewer

Reply:
You'll want to develop the docker and OCP configurations together, since they parallel each other.